tools/bs_parser_hevc/tools/hevc_fei_extractor/src/main.cpp

```diff
@@ -67,7 +67,7 @@ class FileHandler
             m_errorSts = true;// Error sts
         }
 
-        if ((m_pFile = fopen(fileName, mode)) == nullptr) {
+        if (fileName && (m_pFile = fopen(fileName, mode)) == nullptr) {
```

Comment:
filename is checked at line 60 above. No need to check again

Reply:
But we do not exit the function after check. If the file name is empty, we still try to open the file.

Reply:
Fixed